schema/artwork/index.js

```diff
@@ -439,6 +439,16 @@ const ArtworkType = new GraphQLObjectType({
       manufacturer: markdown(),
       series: markdown(),
       meta: Meta,
+      related: {
+        type: new GraphQLList(Artwork.type),
+        args: {
```

Comment:
It seems to work though? https://github.com/artsy/gravity/blob/a385e1e2b3fd31c85a30b0b529f30838803ce11b/app/api/v1/related_endpoint.rb#L277